lua/galaxyline/provider_diagnostic.lua

```diff
@@ -21,7 +21,9 @@ local function get_nvim_lsp_diagnostic(diag_type)
     local count = 0
 
     for _, client in ipairs(active_clients) do
-       count = count + lsp.diagnostic.get_count(api.nvim_get_current_buf(),diag_type,client.id)
+       local opts = { severity = diag_type }
+       opts.namespace = vim.diagnostic.get_namespace(client.id)
+       count = count + vim.diagnostic.get(api.nvim_get_current_buf(),opts)
```

Comment:
This patch gives arithmetic errors when displaying the diagnostics at the galaxystatus line.

I may be wrong, but aren't you here adding a scalar `count` to a table from `vim.diagnostic.get()`.

Reply:
I've fixed this in my fork if you're interested.
https://github.com/justinhj/galaxyline.nvim/blob/main/README.md